### DISCUSSION TITLE
What is the deal with pageProps/data

### DISCUSSION THREAD

**Discussion by tlancina** [vikejs/vike#1426] [Discussion]
I just started a new `react-ts` project and it has a `pageProps` definition and uses that in both onRenderHtml and onRenderClient, is it fair to say that based on [the TODO](https://github.com/vikejs/vike/blob/d07fbe827118677177ef1226afb5ba23fc86fd43/vike/TODO#L1) I should swap those out with `data`?

**Comment by tlancina** [comment]
Seems to work just fine! I think the docs/examples might just be in flux. Really liking Vike so far, especially coming from Next.

**Reply by brillout** [reply]
Indeed, the docs now recommend https://vike.dev/data instead of `pageProps`. The boilerplates are updated as well.

I'm glad you're enjoying vike. Would your company be up for [sponsoring](https://github.com/vikejs/vike/issues/1350)?
